### ISSUE TITLE
invert colors?

### ISSUE THREAD

**Issue by l3land** [negrel/wooz#14]
Hi! Many users who need magnification are also photophobic (are pained/blinded by the glare of a bright white web page e.g.) and need to be able to invert colors. MATE, GNOME, et al have accessibility features that do this, but not compositors like niri et al. So it would be lovely to have WOoz bring this functionality! I am a nontechnical dude so I cannot offer to write this code myself, just throwin it out there :)

or if anyone has advice on how to get invert colors in a compositor like Niri i'm all ears! :)
